docs/source/intro/install_conda.rst

```diff
@@ -43,30 +44,22 @@ This is the recommended method for users on shared systems and systems where the
 
    .. admonition:: Further Reading
        :class: seealso title-icon-read-more
-
+   
        - `Creating a Database Cluster <https://www.postgresql.org/docs/12/creating-cluster.html>`__.
        - `Starting the Database Server <https://www.postgresql.org/docs/12/server-start.html>`__.
+   
+
 
    Then, start the RabbitMQ server:
 
    .. code-block:: console
 
        (aiida) $ rabbitmq-server -detached
-
-   Finally, start the AiiDA daemon(s):
-
-   .. code-block:: console
-
-       (aiida) $ verdi daemon start 2
-
-   .. important::
-
-        The verdi daemon(s) must be restarted after a system reboot.
-
-   .. tip::
-
-       Do not start more daemons then there are physical processors on your system.
-
+   
+   .. important:: 
+   
+        The services started this way will use the default ports on the machine. Conflicts may happen if there are more than one user running AiiDA this way on the same machine, or you already have the server running in a system-wide installation. To get around this issue, you can explicitly define the ports to be used.
```

Comment:
We should generally use *one sentence per line*.
```suggestion
        The services started this way will use the default ports on the machine.
        Conflicts may happen if there are more than one user running AiiDA this way on the same machine, or you already have the server running in a system-wide installation.
        To get around this issue, you can explicitly define the ports to be used.
```

Reply:
Sure. I will make it one sentence per line.